migrator/alias.go

```diff
@@ -50,36 +51,43 @@ func (m *Migrator) migrateCurrentAlias() (err error) {
 	return
 }
 
-func (m *Migrator) migrateCurrentReverseAlias() (err error) {
-	var ids []string
-	ids, err = m.dmIN.DataDB().GetKeysForPrefix(utils.REVERSE_ALIASES_PREFIX)
-	if err != nil {
-		return err
-	}
-	for _, id := range ids {
-		idg := strings.TrimPrefix(id, utils.REVERSE_ALIASES_PREFIX)
+// func (m *Migrator) migrateCurrentReverseAlias() (err error) {
```

Comment:
If we don't longer need this, please remove from sources instead of  leaving it commented.